sys-apps/smcipmitool/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>ck+gentoo@bl4ckb0x.de</email>
+		<name>Conrad Kostecki</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<use>
+		<flag name="bundled-jre">Use bundled Java JRE instead of system JRE.</flag>
```

Comment:
Is there a very good reason to use either of that? If not, please unbundle unconditionally.

Reply:
From point of "working" not. I only thought, that in case, if there would something go wrong with a newer java version, you can use the shipped one. It's working for me fine with the current java version in portage. So remove it and used always system java?

Reply:
Please do. Bundled libraries are a major PITA for security, so better not to add it if unnecessary.

Reply:
Allright, I've pushed a new version, which uses only system JRE and system stunnel, as it works for me.